src/anyvlm/anyvar/base_client.py

```diff
@@ -21,6 +22,21 @@ class AnyVarClientConnectionError(AnyVarClientError):
 class BaseAnyVarClient(abc.ABC):
     """Interface elements for an AnyVar client"""
 
+    @abc.abstractmethod
+    def get_registered_allele_expression(
```

Comment:
I think "expression" is what we have and "allele" is what we want

```suggestion
    def get_registered_allele(
```